pkg/document/gc_test.go

```
@@ -22,9 +22,16 @@ import (
 
 	"github.com/stretchr/testify/assert"
 
+	"os"
+
+	"google.golang.org/protobuf/proto"
+
+	api "github.com/yorkie-team/yorkie/api/yorkie/v1"
```

golangci-lint (2.11.4)

[error] File is not properly formatted

(gofmt)